packages/blockchain-extension/cucumber/steps/gateway.steps.ts

```diff
@@ -152,7 +151,7 @@ module.exports = function(): any {
                 openFileNameArray.includes(filePath).should.be.true;
 
                 // Get the smart contract metadata
-                const connection: IFabricClientConnection = FabricConnectionManager.instance().getConnection();
+                const connection: IFabricGatewayConnection = FabricConnectionManager.instance().getConnection();
```

Comment:
Is FabricConnectionManager ever going to return anything other than a (I)FabricGatewayConnection?
If not - does it make sense to rename FabricConnectionManager to FabricGatewayManager?